app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :likes
+  has_many :comments
```

Comment:
# Optional:

**Maybe adding the foreign keys would help. example:**

- has_many :posts, foreign_key: 'author_id'
- has_many :comments, foreign_key: 'author_id'
- has_many :likes, foreign_key: 'author_id'